documentation/src/main/asciidoc/topics/data_container/proc_configuring_eviction.adoc

```diff
@@ -0,0 +1,14 @@
+[id='configuring_eviction_{context}']
+= Configuring {brandname} to Evict Entries from Memory
+
+. Set a maximum limit for entries in the cache using either a `COUNT` or `MEMORY` measurement.
+. Set a storage type that defines how {brandname} stores entries, either as objects or `bytes[]` in or outside the Java heap.
+
+Declaratively::
```

Comment:
I believe it would be worth it to have two procedures, one for each way of configuring (I as a person with no knowledge of Infinispan have no clue how to do either of those or where to put them). If you're assuming this knowledge from the reader, this info should be in the introduction.